src/monitors/prometheus.rs

```diff
@@ -24,6 +24,10 @@ lazy_static! {
         Mutex::new(ObClientRegistry::default());
 }
 
+pub fn obkv_get_registry() -> MutexGuard<'static, ObClientRegistry> {
+    OBKV_CLIENT_REGISTRY.lock().unwrap()
+}
```

Comment:
I guess there is no need to expose the ObClientRegistry, and what the user needs is just the metrics content. So how about putting it like this:
```
pub fn dump_metrics() -> Result<String, ??> {
    let registry = OBKV_CLIENT_REGISTRY.lock().unwrap();
    let mut buffer = String::new();
    encode(&mut buffer, &registry)?;
    
    Ok(buffer)
}
```